### PR TITLE
CPR fix/buff

### PR DESCRIPTION
**Pull request by ATH1909**
## About The Pull Request

This PR allows you to perform CPR on someone who has over 50 oxyloss damage, even if they're not in crit (they still have to be prone, though).

## Why It's Good For The Game

If your oxyloss damage goes 50, you fall unconscious until it drops below 50. The problem here comes when you get healed out of crit but still have, say, 60 oxyloss damage. You're unconscious from the oxyloss damage and aren't regenning it yourself, but your doctor can't use CPR on you because you're not in crit. What ends up happening is that, if you don't have oxyloss healing meds on you, you have to _beat your patient back into crit_, use CPR on them to heal their oxyloss damage, then heal the brute/burn damage that you dealt to them to put them in crit. This is a pretty arbitrary/immersion breaking thing (why can I use CPR on a critted person, but not on an unconscious person?), so I've made a one line change to CPR to allow it to be used on people who are over the oxyloss unconsciousness threshold.

## Changelog
:cl: ATHATH
fix: You can now use CPR not only on people who are in crit, but also on people who are unconscious due to oxygen damage.
/:cl:

### CODE REVIEW

**Comment by ATH1909** [comment]
I could make this check for if your patient is unconscious instead of if they have >50 oxyloss damage and are unable to stand, but I think that doing that could result in some weird/immersion-breaking stuff happening.

**Comment by kriskog** [comment]
Meet me in dorms 4 for some cpr

**Comment by Krysonism** [comment]
If you don't have oxymeds you should be forced to let oxy damage heal by itself.

>This is a pretty arbitrary/immersion breaking thing (why can I use CPR on a critted person, but not on an unconscious person?)

That is because they are breathing by themselves at that point.

**Comment by ExcessiveUseOfCobblestone** [comment]
what is the fix

File: code/modules/mob/living/carbon/human/species.dm [hunk 1]
```diff
@@ -1158,7 +1158,7 @@ GLOBAL_LIST_EMPTY(roundstart_races)
 	return
 
 /datum/species/proc/help(mob/living/carbon/human/user, mob/living/carbon/human/target, datum/martial_art/attacker_style)
-	if(!((target.health < 0 || HAS_TRAIT(target, TRAIT_FAKEDEATH)) && !(target.mobility_flags & MOBILITY_STAND)))
+	if(!((target.health < 0 || HAS_TRAIT(target, TRAIT_FAKEDEATH) || target.getOxyLoss() > 50) && !(target.mobility_flags & MOBILITY_STAND)))
```
Comment (ShizCalev) [inline code/modules/mob/living/carbon/human/species.dm]: >= imo

**Review by ExcessiveUseOfCobblestone — CHANGES_REQUESTED** [review]
^ agreed

**Comment by ExcessiveUseOfCobblestone** [comment]
getting their first kiss

**Comment by PKPenguin321** [comment]
Just because a patient isn't in crit doesn't mean they can breathe on their own
That said, it would make sense if the only CPR conditions were nobreath and that they're prone

**Comment by PKPenguin321** [comment]
Pls merge this I really want it

**Comment by PKPenguin321** [comment]
@ATH1909 lad, please just address the 1 line change review

**Pull request closed without merging**